### DISCUSSION TITLE
Digitalocean Enjoy $200 in platform credit for 1 year or 60days?

### DISCUSSION THREAD

**Discussion by imsatyam2610** [community/community#67253] [GitHub Education] [answered]
### Hi everyone,

The benefit of Digitalocean cloud hosting is the credit only for 60 days or 1 year that get from GitHub Student Developer Pack

**Comment by hafiz-muhammad** [comment, marked as answer]
It  is  valid for 1 year. [GitHub Student Developer Pack Benefits Page](https://education.github.com/pack?utm_source=github+digitalocean)

![2023-09-16_02-44](https://github.com/community/community/assets/109435420/008f1844-4c84-4bd0-bb9b-7c64b88b50c4)

**Reply by wolf1994722** [reply]
What are the preferential treatment options?
Isn't it too expensive?

**Comment by imsatyam2610** [comment]
Getting this error while redeeming the Digitalocean GitHub student developer pack benefit: "Promotion cannot be applied to this account". Even this is new account and also authorized access of GitHub to Digitalocean.

![Screenshot 2023-09-23 143054](https://github.com/community/community/assets/62741901/dd0aec4c-764e-43db-b872-ca2f4a3984b4)

**Reply by JAWalmsley** [reply]
Having same error as above, did you resolve it?

**Reply by risqiikhsani** [reply]
SAME !

**Reply by kamranzafar4343** [reply]
Ensure your GitHub account is verified through GitHub Education, use a new DigitalOcean account, and check for proper authorization; if issues persist, contact support for both platforms.

**Reply by mradzikowski** [reply]
same!

**Reply by shivpratapsingh111** [reply]
Did anyone got it resolved?

**Reply by p4r7h** [reply]
same not working

**Reply by SavinDevelop** [reply]
It doesn't work for me either

**Reply by SavinDevelop** [reply]
People from the future, if you have already created an account based on github authentication, then check the calculations team, added without my intervention

**Reply by Plusero** [reply]
Same here, and haven't solved it yet. I think it is because the 60 days credits are initialized first and it is blocking the 1 year credits.

**Reply by mateusloubach** [reply]
anyone got this solved?

By reading maybe 20 pages of github community discussions, I'm reaching the conclusion github and digitalocean is avoiding talking about this. It's the same error for everyone complaining and neither supports wants to solve the issue.

**Comment by artyushinav** [comment]
кредит имеется в виду прям кредит который в будущем нужно будет отдавать или можно потратить на свои моменты и не надо будет платить?

**Comment by VESoft** [comment]
<h1>Claim Your $200 Credit Coupon - Valid for 2 Months!</h1>
    <p>This exclusive $200 coupon can be used to create any type of server infrastructure on DigitalOcean. Enjoy high-performance cloud solutions with flexibility to scale and manage your projects efficiently.</p>
    <p>Use it to launch applications, build powerful databases, deploy web servers, and more. Start your journey with DigitalOcean today!</p>
    <a href="https://www.digitalocean.com/?refcode=1baaffe2f0cb&utm_campaign=Referral_Invite&utm_medium=Referral_Program&utm_source=badge">
        <img src="https://web-platforms.sfo2.cdn.digitaloceanspaces.com/WWW/Badge%203.svg" alt="DigitalOcean Referral Badge" />
    </a>
    <p><strong>Click the badge above to claim your credit and start building!</strong></p>
